index.html

```diff
@@ -0,0 +1,390 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>David Underwood</title>
+  </head>
+  <body>
+    <header>
+      <a href="/" class="header-logo">
+        <h3>Welcome</h3>
+      </a>
+      <nav>
+        <img class="union" src="Union.png" alt="menu link" />
+      </nav>
+    </header>
+
+    <div class="greet">
```

Comment:
- [ ] Good job on your desktop version UI. it looks good 😄. I just noticed a horizontal scroll bar on your desktop view which is not a good user experience. Kindly fix it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/187987484-e336f456-8a83-4039-b666-e1ec3172606b.png)

Reply:
@vickymarz thanks for the comment but I don't have this horizontal scroll bar on my local so I am not sure what I should do?

Reply:
@ Try and inspect the webpage from your end first

Reply:
ok

Reply:
@vickymarz I added overflow-x: hidden to my body I hope that fixes it because I can't check if it works because I don't have the problem